gio/src/subclass/file_enumerator.rs

```diff
@@ -0,0 +1,154 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
```

Comment:
Can you pull this into a separate PR as well? so you only keep GFile for this one.